holmes/plugins/toolsets/kafka.py

```
@@ -818,7 +938,363 @@ def get_parameterized_one_liner(self, params: Dict) -> str:
+                "configs": configs,
+            }
+
+            return StructuredToolResult(
+                status=StructuredToolResultStatus.SUCCESS,
+                data=yaml.dump(result),
+                params=params,
+            )
+        except Exception as e:
+            error_msg = (
+                f"Failed to describe configs for {resource_type} '{resource_name}' "
+                f"on cluster '{kafka_cluster_name}': {str(e)}"
```

Ruff (0.15.9)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)